src/documentation/sass/ComponentPage.scss

```diff
@@ -199,10 +199,18 @@ $dontColor: #A61E22;
 
 .ComponentPage-statesSection,
 .ComponentPage-methodsSection {
-  overflow: scroll;
+  overflow: auto;
 }
 
 
+.ms-TableData {
+  tbody {
+    td {
```

Comment:
nit: since there's just one property change, rather than nesting, this selector could be shortened to `tbody td {...}`